### DISCUSSION TITLE
Tailwind version 3 doesn't work for classname autocompletion in phpstorm

### DISCUSSION THREAD

**Discussion by zinminnhtun** [tailwindlabs/tailwindcss#7184] [Help] [answered]
Tailwind version 3 doesn't work for class name auto-completion in Phpstorm.But version 2 works.How can I fix for updated tailwind v3 to get auto-completion?

**Comment by KINGER404** [comment, marked as answer]
Try to update PhpStorm to latest version
<img width="300" alt="изображение" src="https://user-images.githubusercontent.com/53098109/150883394-5485a838-587a-45a9-9324-eaa33df987d1.png">

**Reply by zinminnhtun** [reply]
Thank you for help.Yeah phpstorm version 2021.3.1 works.
